client/analytics/settings/historical-data/status.js

```diff
@@ -0,0 +1,37 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { applyFilters } from '@wordpress/hooks';
+import moment from 'moment';
+import { Spinner } from '@wordpress/components';
+
+/**
+ * WooCommerce dependencies
+ */
+import { useFilters } from '@woocommerce/components';
+
+const HISTORICAL_DATA_STATUS_FILTER = 'woocommerce-import-status';
```

Comment:
Awesome! 🎉 Small nitpick:  Can we use the `woocommerce_admin_` prefix to keep consistent with the other filters?

```suggestion
const HISTORICAL_DATA_STATUS_FILTER = 'woocommerce_admin_import_status';
```

Reply:
Done in b3058b120a82e16862f51f69adfab27a68bf6e4b. :+1: